### PR TITLE
separate datadog-builder from bastion role

### PR DESCRIPTION
**Pull request by loafyloaf**
Signed-off-by: matthew wagoner <zxkuqyb@gmail.com>

### CODE REVIEW

File: dd-builder.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - role: logrotate
+      logrotate_configs:
+        - name: datadog-builder
+          path: /var/log/datadog-builder/*log
+          options:
+            - compress
+            - missingok
+            - rotate 30
+            - daily
+            - notifempty
```
Comment (gandelman-a) [inline dd-builder.yml:19]: So as-is, nothing is setup to actually run this playbook in the same way we  do bastion.yml or install-ci.yml.  Was this intentional?

**Comment by gandelman-a** [comment]
Without a commit msg its kinda hard to determine the intent of this?  Its just moving the datadog-builder role off of the bastion into its own group, but results in that stuff not running anymore.

**Review by gandelman-a — CHANGES_REQUESTED** [review]

**Comment by loafyloaf** [comment]
so, at the time, datadog-builder was running on two bastions - contrasjc and opentechsjc.  While this shouldn't really be a problem, the monitors.yml file was not getting updated on contrasjc, so when 'new' bastion created a monitor, 'old' bastion would delete it.  This is a bit moot now, seeing as how contrasjc is gone.  But perhaps we may get a second cloud in the future...

(aplogies for the lacking commit msg)

File: dd-builder.yml (hunk 1 above)
Comment (ConsoleCatzirl) [reply dd-builder.yml:19]: Is the idea to create a new ansible-runner cron for this playbook?

**Comment by mlangbehn** [comment]
Per discussion with @loafyloaf in IRC, this pull request is no longer needed. Closing accordingly.

**Pull request closed without merging**